packages/material/scss/scrollview/_theme.scss

```diff
@@ -0,0 +1,34 @@
+@import "~@progress/kendo-theme-default/scss/scrollview/theme";
+
+@include exports("scrollview/theme/material") {
+    $scrollview-text-opacity: .4 !default;
+    $scrollview-padding-y: 10px !default;
+    $scrollview-padding-x: $padding-y !default;
+
+    .k-scrollview-next,
+    .k-scrollview-prev {
+        span {
+            color: $base-text;
```

Comment:
The same effect can be achieved without additional styles, by setting `$scrollview-navigation-color: $base-text` before importing the default theme.